_extras/demo_lessons.md

```diff
@@ -20,11 +20,11 @@ If the chosen episode is very short and finishes in less than five minutes, the
+| [Data Organization in Spreadsheets](https://datacarpentry.org/spreadsheet-ecology-lesson/) |  any other episode | none ||
+| [Data Cleaning with OpenRefine](https://datacarpentry.org/OpenRefine-ecology-lesson/) |  [Working with OpenRefine](https://datacarpentry.org/OpenRefine-ecology-lesson/01-working-with-openrefine) | any other episode | The intro episode has no live coding. <br/> Other episodes have dependencies. |
 | [Data Analysis and Visualization in R](https://datacarpentry.org/R-ecology-lesson/) |  * Introduction to R<br/>* Starting with Data | * [Before We Start](http://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html)<br/> * [Manipulating Data](https://datacarpentry.org/R-ecology-lesson/03-dplyr.html)<br/> * [Data Visualization with ggplot2](https://datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html)<br/> * [SQL Databases and R](https://datacarpentry.org/R-ecology-lesson/05-r-and-databases.html)<br/>| [Before We Start](http://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html) has no live coding. <br/> Manipulating Data and [Data Visualization with ggplot2](https://datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html) have dependencies. <br/> SQL and `RSQLite` are taught less frequently. |
-| [Data Management with SQL](http://www.datacarpentry.org/sql-ecology-lesson/) | any episode | [Databases using SQL](http://www.datacarpentry.org/sql-ecology-lesson/00-sql-introduction/) | Live coding starts in the middle of the episode to avoid. | 
-| [Data Analysis and Visualization in Python](http://www.datacarpentry.org/python-ecology-lesson/) |  * [Short Introduction to Programming](https://datacarpentry.org/python-ecology-lesson/01-short-introduction-to-Python/)<br/> * [Starting with Data](https://datacarpentry.org/python-ecology-lesson/02-starting-with-data/)<br/> * [Indexing, Slicing, and Subsetting](https://datacarpentry.org/python-ecology-lesson/03-index-slice-subset/)<br/> * [Data Types and Formats](https://datacarpentry.org/python-ecology-lesson/04-data-types-and-format/)<br/> * [Combining DataFrames with Pandas](https://datacarpentry.org/python-ecology-lesson/05-merging-data/)<br/> * [Data Workflows and Automation](https://datacarpentry.org/python-ecology-lesson/06-loops-and-functions/) | any episode | [Before we Start](https://datacarpentry.org/python-ecology-lesson/00-before-we-start/) has no live coding. <br/> Later episodes have dependencies. | 
+| [Data Management with SQL](http://www.datacarpentry.org/sql-ecology-lesson/) | any other episode | [Databases using SQL](http://www.datacarpentry.org/sql-ecology-lesson/00-sql-introduction/) | Live coding starts in the middle of the episode to avoid. | 
```

Comment:
```suggestion
| [Data Management with SQL](http://www.datacarpentry.org/sql-ecology-lesson/) | any episode | [Databases using SQL](http://www.datacarpentry.org/sql-ecology-lesson/00-sql-introduction/) | Live coding starts in the middle of the episode to avoid. | 
```